webots_ros2_examples/webots_ros2_examples/example_controller.py

```diff
@@ -20,25 +20,31 @@
 import rclpy
 
 from std_msgs.msg import Float64
+from geometry_msgs.msg import Twist
 
 
 class ExampleController(WebotsNode):
 
     def __init__(self, args):
         super().__init__('example_controller', args)
-        self.sensorTimer = self.create_timer(0.001 * self.timestep, self.sensor_callback)
+        self.sensorTimer = self.create_timer(0.001 * self.timestep,
+                                             self.sensor_callback)
         self.leftMotor = self.robot.getMotor('motor.left')
         self.rightMotor = self.robot.getMotor('motor.right')
         self.leftMotor.setPosition(float('inf'))
         self.rightMotor.setPosition(float('inf'))
         self.leftMotor.setVelocity(0)
         self.rightMotor.setVelocity(0)
-        self.motorService = self.create_service(SetDifferentialWheelSpeed, 'motor',
-                                                self.motor_callback)
+        self.motorMaxSpeed = self.leftMotor.getMaxVelocity()
+        self.motorService = self.create_service(SetDifferentialWheelSpeed,
+                                                'motor', self.motor_callback)
         self.sensorPublisher = self.create_publisher(Float64, 'sensor', 10)
         # front central proximity sensor
         self.frontSensor = self.robot.getDistanceSensor('prox.horizontal.2')
         self.frontSensor.enable(self.timestep)
+        self.cmdVelSubscriber = self.create_subscription(Twist, 'motor',
```

Comment:
```suggestion
        self.cmdVelSubscriber = self.create_subscription(Twist, 'cmd_vel',
```
According to the row wiki (http://wiki.ros.org/diff_drive_controller), the standard name for this subscriber is `cmd_vel`.
This is also the case in our reference implementation (ROS2 for the turtlebot3):
https://github.com/ROBOTIS-GIT/turtlebot3/blob/ros2/turtlebot3_node/src/turtlebot3.cpp#L320